open_humans/templates/account/signup-modal.html

```diff
@@ -11,24 +11,32 @@ <h4 class="modal-title">Create an Open Humans account</h4>
       </div>
 
       <div class="modal-body">
-        {% include 'account/signup-form.html' %}
+        <div class="col-md-auto">
```

Comment:
In this case the div doesn't seem like this has any purpose. There's no row/column grid in the original version of this template, and this class is meaningless...